README.md

```diff
@@ -44,4 +50,4 @@ allows you to decrypt and inspect your backups.
 It can also re-encrypt them.
 
 ## License
-This application is available as open source under the terms of the [Apache-2.0 License](https://opensource.org/licenses/Apache-2.0).
+This application is available as open source under the terms of the [Apache-2.0 License](https://opensource.org/licenses/Apache-2.0). However, some elements are being licensed under AGPL-3.0-or-later and CC0-1.0. For accurate information, please check individual files.	
```

Comment:
The `AGPL-3.0-or-later` part should probably be clarified according to the answer of the lawyer. Also since the license texts are now part of the repo, we can directly link to them (instead of to opensource.org)?

Reply:
The language of this paragraph was updated.